stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -615,8 +615,12 @@ internal class ChannelControllerImpl(
                 var attachment: Attachment = it
                 if (it.upload != null) {
                     val result = uploadAttachment(it, attachmentTransformer)
+                    attachment.uploadComplete = result.isSuccess
```

Comment:
As I see it's serialized field obtained from BE. Why do you set the value manually here?

Reply:
This information is not generated in the backend. I'm sending the information about the completeness of the upload so we can get it from the backend in a latter moment.